src/org/labkey/test/util/DataRegionTable.java

```diff
@@ -274,17 +272,17 @@ public void assertPaginationText(int firstRow, int lastRow, int totalRows)
     }
 
     /**
-     * @deprecated Renamed. Use {@link #getRowIndex(String, String)}
+     * @deprecated Renamed. Use {@link #getRowIndex(CharSequence, String)}
      */
     @Deprecated
     public int getRow(String columnLabel, String value)
     {
         return getRowIndex(columnLabel, value);
     }
```

Comment:
Given our previous discussion, how much work would it be to remove this deprecated method?

Reply:
Not huge; four lines across four more repositories